department-of-veterans-affairs/caseflow
# Add helper to generate AWS filter log command and URL

Related to #15649

### Description

 * Add a helper class to make it easier to find logs for any particular job

### Testing Plan

1. Find the message ID of a job in UAT that has happened within the last day. You can browse the [UAT logs](https://console.amazonaws-us-gov.com/cloudwatch/home?region=us-gov-west-1#logsV2:logs-insights$3FqueryDetail$3D$257E$2528end$257E0$257Estart$257E-86400$257EtimeType$257E$2527RELATIVE$257Eunit$257E$2527seconds$257EeditorString$257E$2527fields*20*40timestamp*2c*20*40message$257EisLiveTail$257Efalse$257EqueryId$257E$25274c6942d6-b218-4958-95b1-97724144fdc2$257Esource$257E$2528$257E$2527dsva-appeals-certification-uat*2fopt*2fcaseflow-certification*2fsrc*2flog*2fcaseflow-certification-sqs-worker.out$2529$2529) and look for for a string that has a UUID at the end of it: `ActiveJob/EndProductSyncJob/caseflow_uat_low_priority/3fbd0884-4f00-4d17-a866-517eec72a8f`. The UUID is the message ID that `AwsJobLogHelper::JobInfo` accepts. This ID will be exposed to Sentry so you don't have to go looking for the message ID in the future.
2. Open a Rails console, and run:
    ```
    puts AwsJobLogHelper::JobInfo.new(JobClass, 'message_id').url
    ```
3. Paste the URL into your browser, click the `Run Query` button, and ensure you're seeing relevant logging
4. Open a Rails console and run
    ```
    puts AwsJobLogHelper::JobInfo.new(JobClass, 'message_id').command
    ```
5. Run `issue_mfa`, then try running the command that was outputted in Rails console
6. After a few seconds, run the second command it tells you to run, and ensure you're seeing relevant logging info

### Code Documentation Updates
